src/collections/_documentation/accounts/pricing.md

```diff
@@ -50,22 +48,262 @@ You can set a maximum monthly on-demand bill amount by setting an on-demand spen
+      <td>$0.000300</td>
+      <td>$0.000390</td>
+    </tr>
+    <tr>
+      <td>>2M-10M</td>
+      <td>1M</td>
+      <td>$0.000150</td>
+      <td>$0.000195</td>
+      <td>$0.000300</td>
+      <td>$0.000390</td>
+    </tr>
+    <tr>
+      <td>>10M-20M</td>
+      <td>1M</td>
+      <td>$0.000120</td>
+      <td>$0.000156</td>
+      <td>$0.000144</td>
+      <td>$0.000187</td>
+    </tr>
+    <tr>
+      <td>>20M-50M</td>
+      <td>2M</td>
+      <td>$0.000110</td>
+      <td>$0.000143</td>
+      <td>$0.000132</td>
+      <td>$0.000172</td>
+    </tr>
+    
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+    <td>
+      <strong>TRANSACTIONS </strong>
+    </td>
+    <td>
+      <strong> </strong>
+    </td>
+      <td>
+        <strong>Team Plan</strong>
+      </td>
+      <td>
+        <strong></strong>
+      </td>
+      <td>
+        <strong>Business Plan</strong>
+      </td>
+      <td>
+        <strong></strong>
+      </td>
+    </tr>
+    <tr>
+      <td><i>Volume</i></td>
+      <td><i>Prepaid Quota</i></td>
+      <td><i>Prepaid</i></td>
+      <td><i>On-Demand</i></td>
+      <td><i>Prepaid</i></td>
+      <td><i>On-Demand</i></td>
+    </tr>
+    <tr>
+      <td>>100k-250k</td>
+      <td>150k</td>
+      <td>$0.000100</td>
+      <td>$0.000130</td>
+      <td>$0.000300</td>
+      <td>$0.000390</td>
+    </tr>
+    <tr>
+      <td>>250k-500k</td>
+      <td>250k</td>
+      <td>$0.000058</td>
+      <td>$0.000075</td>
+      <td>$0.000178</td>
+      <td>$0.000231</td>
+    </tr>
+    <tr>
+      <td>>500k-4.5M</td>
+      <td>500k</td>
+      <td>$0.000052</td>
+      <td>$0.000068</td>
+      <td>$0.000130</td>
+      <td>$0.000169</td>
+    </tr>
+    <tr>
+      <td>>4.5M-5M</td>
+      <td>500k</td>
+      <td>$0.000046</td>
+      <td>$0.000060</td>
+      <td>$0.000092</td>
+      <td>$0.000120</td>
+    </tr>
+    <tr>
+      <td>>5M-10M</td>
+      <td>1M</td>
+      <td>$0.000046</td>
+      <td>$0.000060</td>
+      <td>$0.000092</td>
+      <td>$0.000120</td>
+    </tr>
+    <tr>
+      <td>>10M-20M</td>
+      <td>2M</td>
+      <td>$0.000044</td>
+      <td>$0.000057</td>
+      <td>$0.000077</td>
+      <td>$0.000100</td>
+    </tr>
+    <tr>
+      <td>>20M-50M</td>
+      <td>5M</td>
+      <td>$0.000042</td>
+      <td>$0.000055</td>
+      <td>$0.000074</td>
+      <td>$0.000096</td>
+    </tr>
+    
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+    <td>
+      <strong>ATTACHMENTS </strong>
+    </td>
+    <td>
+      <strong> </strong>
+    </td>
+      <td>
+        <strong>Team/Business Plan</strong>
+      </td>
+      <td>
+        <strong></strong>
+      </td>
+    </tr>
+    <tr>
+      <td><i>Volume</i></td>
+      <td><i>Prepaid Quota</i></td>
+      <td><i>Prepaid</i></td>
+      <td><i>On-Demand</i></td>
+    </tr>
+    <tr>
+      <td>25-100 (GB)</td>
+      <td>25 GB</td>
+      <td>$0.25</td>
+      <td>$0.325</td>
+    </tr>
+    <tr>
+      <td>>100-10k (GB)</td>
```

Comment:
nit: 100 (GB) - 10 (TB)

Reply:
Oh, good catch. Thanks.

Reply:
ahh i'm sorry about this- heard back from finance we're switching to using GB only for attachments in customer facing material so what you had originally was correct

Reply:
np. Will fix right now.